recipe/install-liblightgbm.sh

```diff
@@ -0,0 +1,50 @@
+# *****************************************************************
+#
+# Licensed Materials - Property of IBM
+#
+# (C) Copyright IBM Corp. 2020. All Rights Reserved.
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+#
+# *****************************************************************
+#!/bin/bash
+
+export CMAKE_PREFIX_PATH=$PREFIX
+
+mkdir -p ${SRC_DIR}/build
+cd ${SRC_DIR}/build
+
+BUILD_OPTION=""
+# Determine Architecture
+ARCH="$(arch)"
+
+if [[ $build_type == "cuda" ]]
+then
+    BUILD_OPTION="-DUSE_CUDA=ON -DCMAKE_CUDA_COMPILER=${CUDA_HOME}/bin/nvcc -DCMAKE_CUDA_HOST_COMPILER=${CXX}"
+
+    # Create symlinks of cublas headers into CONDA_PREFIX
+    mkdir -p $CONDA_PREFIX/include
+    find /usr/include -name cublas*.h -exec ln -s "{}" "$CONDA_PREFIX/include/" ';'
+    export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -I${CUDA_HOME}/include -I${CONDA_PREFIX}/include"
+
+    if [[ "${ARCH}" == 'ppc64le' ]]
+    then 
+        export CUDA_ARCH_FLAGS="3.7 6.0 7.0 7.5 8.0"
+    else
+        export CUDA_ARCH_FLAGS="3.7 5.2, 6.0 6.1 7.0 7.5 8.0"
+    fi
+    
+fi
+
+if [[ $mpi_type == 'openmpi' ]]
```

Comment:
Right now, the two types of MPI that we support are `openmpi` and `system`. The `system` type assumes that MPI is already installed on the system, and we would still want to build with MPI in that case. Eventually we probably want an `mpi_type` of `none`. In that case, we would want to turn MPI off.

Reply:
Yes, I'd thought of this too but not sure if mpi_type being None is supported yet or not.

Reply:
Looks like it is working. Thanks @bnemanich for pointing this.